mappings/net/minecraft/item/Item.mapping

```diff
@@ -68,7 +85,9 @@ CLASS net/minecraft/class_1792 net/minecraft/item/Item
 	METHOD method_7856 isEffectiveOn (Lnet/minecraft/class_2680;)Z
 		ARG 1 state
 	METHOD method_7857 hasRecipeRemainder ()Z
+		COMMENT Checks if this item has a remainder item that should be left behind when crafted.
```

Comment:
```suggestion
		COMMENT Checks if this item has a remainder item that is left behind when used as a crafting ingredient.
```
Correct me if this is used for non-crafting like smelting as well.

Reply:
This actually is called in the brewing stand weirdly enough. If there is a remainder in the brewing stand, then the item is just scattered into the world,